robot/Cumulus/resources/ContactPageObject.py

```diff
@@ -26,14 +25,41 @@ def _is_current_page(self):
         self.selenium.wait_until_location_contains("/view", timeout=60, message="Detail page did not load in 1 min")
         self.selenium.location_should_contain("/lightning/r/Contact/",message="Current page is not a Contact record detail view")
         
-
     def update_field_value(self,field_name,old_value,new_value):
         """Delete the old value in specified field by clicking on delete icon and update with new value"""
         locator=npsp_lex_locators['delete_icon'].format(field_name,old_value)
         self.selenium.get_webelement(locator).click() 
         self.salesforce.populate_lookup_field(field_name,new_value)
-        
-            
+
+    def go_to_related_engagement_actionplans_page(self,customerid):
+
+        """ Navigates to the related engagement plans listing page"""
+        url = self.cumulusci.org.lightning_base_url
+        name = "Action_Plans__r"
+        object_name = "{}{}".format(self.cumulusci.get_namespace_prefix(), name)
+        url = "{}/lightning/r/{}/related/{}/view".format(url,customerid,object_name)
+        self.selenium.go_to(url)
+        self.salesforce.wait_until_loading_is_complete()
+        self.selenium.wait_until_location_contains("/view",timeout=60, message="Page not loaded")
+
+    def perform_action_on_related_item(self,action):
+        """ Identifies the value to delete from the List and chooses delete
+            option from the menu. Confirms the delete action from the confirmation modal
+        """
+        drop_down = npsp_lex_locators['opportunities_dropdown'].format(1)
```

Comment:
Is '1', an index value?

Reply:
yes. There will be only one record from the testexeuction point of view at all times